be2-scala/src/main/scala/ch/epfl/pop/storage/DbActor.scala

```diff
@@ -45,6 +45,21 @@ final case class DbActor(
     }
   }
 
+  @throws[DbActorNAckException]
+  private def writeCreateLao(channel: Channel, message: Message): Unit = {
+    createChannel(channel, ObjectType.LAO)
+    storage.write((storage.DATA_KEY + storage.CREATE_LAO_KEY + channel.toString, message.message_id.toString()))
+    write(Channel.ROOT_CHANNEL, message)
+  }
+
+  @throws[DbActorNAckException]
+  private def readCreateLao(channel: Channel): Option[Message] = {
+    storage.read(storage.DATA_KEY + storage.CREATE_LAO_KEY + channel.toString) match {
```

Comment:
Now that I think about it, the `read` function defined inside `DbActor.scala` should be commented a bit: we should precise that we use this function to read messages except the `CreateLao` ones (smth like this).